hawc/apps/assessment/api/viewsets.py

```diff
@@ -67,7 +71,10 @@ def fields(self, request, format=None):
         cleanup_fields = self.model.TEXT_CLEANUP_FIELDS
         TERM_FIELD_MAPPING = getattr(self.model, "TERM_FIELD_MAPPING", {})
         return Response(
-            {"text_cleanup_fields": cleanup_fields, "term_field_mapping": TERM_FIELD_MAPPING}
+            {
+                "text_cleanup_fields": cleanup_fields,
+                "term_field_mapping": TERM_FIELD_MAPPING,
```

Comment:
I think your local setup is modifying files adding trailing commas - can you revert these changes and maybe disable this feature in your local setup? It makes it hard to review